origamiUROP/oxdna/strand.py

```diff
@@ -42,6 +42,10 @@ def nucleotides(self) -> list:
             nucleotide._strand_index = self.index
         return self._nucleotides
 
+    def add_nucleotide(self, Nucleotide: Nucleotide):
```

Comment:
don't use `Nucleotide` since this is already declared as a class - use `nucleotide`